src/EnergyPlus/DXCoils.cc

```diff
@@ -1230,13 +1231,17 @@ namespace DXCoils {
 				}
 			}
 
-			DXCoil( DXCoilNum ).Twet_Rated( 1 ) = Numbers( 6 );
-			DXCoil( DXCoilNum ).Gamma_Rated( 1 ) = Numbers( 7 );
-			DXCoil( DXCoilNum ).MaxONOFFCyclesperHour( 1 ) = Numbers( 8 );
-			DXCoil( DXCoilNum ).LatentCapacityTimeConstant( 1 ) = Numbers( 9 );
+			//Set minimum OAT for compressor operation
+			DXCoil( DXCoilNum ).MinOATCompressor = Numbers( 6 );
+			if ( NumNumbers < 6 ) DXCoil( DXCoilNum ).MinOATCompressor = minOATCompDXCooling; // input field is after min fields and won't default if field not included 
```

Comment:
@MyOldMopar @mjwitte the new input for min OAT comp in Coil:Cooing:DX:SingleSpeed is in field 15 when min-fields is 14. It seemed like the most logical place. I have this check in place to ensure MinOATCompressor is initialized to the existing default. If someone changes the default in the IDD and does not enter this field, it will still default to -25. If this field is entered and blank, the new user defined default min OAT will be used. Just as we know a field is blank, I would like to know the IDD field default so I can use that default in places like this.